api/lib/error.handler.ts

```diff
@@ -13,6 +13,10 @@ export default function errorHandler(err, req: Request, res: Response, next: Nex
         // remove any sensitive data, such as bearer token from being logged
         delete err.config.headers
     }
+    if (propsExist(err, ['request', '_header'])) {
```

Comment:
Can you test that as well please?

Reply:
There are no tests for this file. Plus this is urgent

Reply:
Unit test added